### ISSUE TITLE
mock.js is missing from 2.0.4 release

### ISSUE THREAD

**Issue by lonnylot** [zoontek/react-native-permissions#380]
# Bug

The new `mock.js` file is missing from v2.0.4.

## Environment info

React native info output:

```bash
info Fetching system and libraries information...
System:
    OS: macOS 10.15.1
    CPU: (4) x64 Intel(R) Core(TM) i5-6267U CPU @ 2.90GHz
    Memory: 5.43 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.11.3 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 23, 28
      Build Tools: 28.0.3, 29.0.0
      System Images: android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.2.1/11B500 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-create-library: 3.1.2
    react-native-log-ios: 1.0.1
```

Library version: 2.0.4

## Steps To Reproduce

1. react-native init demo
2. npm install react-native-permissions

Describe what you expected to happen:

```bash
lonnykapelushnik@Lonnys-MBP rndemo $ npm install react-native-permissions
[...snip]
+ react-native-permissions@2.0.4
added 1 package from 1 contributor and audited 951422 packages in 6.583s
found 4 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

lonnykapelushnik@Lonnys-MBP rndemo $ ls -l node_modules/react-native-permissions/
total 72
-rw-r--r--   1 lonnykapelushnik  staff   1076 Oct 26  1985 LICENSE
-rw-r--r--   1 lonnykapelushnik  staff  23261 Oct 26  1985 README.md
-rw-r--r--   1 lonnykapelushnik  staff    737 Oct 26  1985 RNPermissions.podspec
drwxr-xr-x   4 lonnykapelushnik  staff    128 Dec  2 11:14 android
drwxr-xr-x  38 lonnykapelushnik  staff   1216 Dec  2 11:14 ios
drwxr-xr-x   5 lonnykapelushnik  staff    160 Dec  2 11:14 lib
-rw-r--r--   1 lonnykapelushnik  staff   3179 Dec  2 11:14 package.json
drwxr-xr-x   5 lonnykapelushnik  staff    160 Dec  2 11:14 src
```

**Comment by zoontek** [comment]
@lonnylot My bad, it's a stupid mistake: https://github.com/react-native-community/react-native-permissions/commit/e4a36da88b6b75ade03e9c81190b68f73e39cc69

I just published `2.0.5` with it

**Comment by lonnylot** [comment]
@zoontek No worries and thanks. It's now there.
